index.js

```diff
@@ -61,7 +53,7 @@ module.exports = {
             target: 'serverless'
           }
         `
-      await appendFile('next.config.js', nextConfig)
+      appendFileSync('next.config.js', nextConfig)
```

Comment:
Note: if you want to keep it async while maintaining old Node versions (i.e. without `fs.promise`), you could also use:

```js
const { promisify } = require('util')
const appendFile = promisify(fs.appendFile)
```